docs/workflows/copy.rst

```diff
@@ -47,47 +47,57 @@ depend on the existence of other content units to be "correct" or meaningful. Fo
 #. A Package Group is a group of RPM packages. If the RPM packages that a Package Group contains
    are not present in the same repository, the Package Group is effectively "broken" and won't be
    possible to install correctly on a client system.
+#. A Module consists of many RPM packages (similar to a Package Group). If the module is added to
+   a repository while the packages that its RPMs depend on are not, it may not be possible to
+   install the module on a client system.
 #. A Module can depend on other modules. If those modules are not present in the RPM repo, the
    module will not be installable on a client system.
 #. RPM Packages typically depend on other RPM packages. If a lone RPM package is added to a
    repository without its dependencies, it will potentially not be installable on a client system.
-#. A Module consists of many RPM packages (similar to a Package Group). If the module is added to
-   a repository while the packages that its RPMs depend on are not, it may not be possible to
-   install the module on a client system.
 
-The advanced copy API exists primarily to address these use cases. Unlike the repository
-modification API, when a copy is performed using the RPM copy API it is permitted to additionally
-copy content in the background which you, the user, did not explicitly tell it to copy. For
-example:
+The advanced copy API exists primarily to address these use cases.
+
+In contrast to the repository modification API, when a copy is performed using the RPM copy
+API it is possible for content in addition to that explicitly requested to be copied to the
+destination.  For example:
```

Comment:
I'm not sure I find this paragraph more clear than the original one, with the exception of s/Unlike/In contrast to/

Reply:
I'm not tied to my change here - I'll put it back :)